src/main/resources/db/migration/V1_0_6__update.sql

```diff
@@ -0,0 +1,29 @@
+-- adding tables for multi users access
+
+CREATE TABLE `webusers` (
+  `username` varchar(50) COLLATE utf8mb3_unicode_ci NOT NULL,
+  `password` varchar(255) COLLATE utf8mb3_unicode_ci NOT NULL,
+  `enabled` tinyint(1) NOT NULL,
+  PRIMARY KEY (`username`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_unicode_ci;
+
+CREATE TABLE `webauthorities` (

```

Comment:
suggestion: `web_user_authorities`

Reply:
https://github.com/steve-community/steve/pull/1165/files#r1712671917

Reply:
maybe two tables are not needed? controversial suggestion: i know normalization is good etc. but this is really a small use case. denormalize and inject authorities into web_user as a column?

mysql supports [json](https://dev.mysql.com/doc/refman/8.4/en/json.html). this column could be a json array of string values.